internal/website/content/howto/blob/_index.md

```diff
@@ -9,3 +9,246 @@ providers and accessing them via HTTP. These guides show how to work with
 blobs in the Go CDK.
 
 <!--more-->
+
+## Opening a Bucket {#opening}
+
+The first step in interacting with unstructured storage is connecting to your
+storage provider. Every storage provider is a little different, but the Go CDK
+lets you interact with all of them using the [`*blob.Bucket` type][].
+
+[`*blob.Bucket` type]: https://godoc.org/gocloud.dev/blob#Bucket
+
+The easiest way to open a blob is using [`blob.OpenBucket`][] and a URL
+pointing to the blob, making sure you ["blank import"][] the driver package to
+link it in. See [Concepts: URLs][] for more details. If you need
+fine-grained control over the connection settings, you can call the constructor
+function in the driver package directly (like `s3blob.OpenBucket`). This guide
+shows how to use both forms for each provider.
+
+Once you have opened a bucket for the storage provider you want, you can
+store and access data from it using the standard Go I/O patterns.
+
+[`blob.OpenBucket`]:
+https://godoc.org/gocloud.dev/blob#OpenBucket
+["blank import"]: https://golang.org/doc/effective_go.html#blank_import
+[Concepts: URLs]: {{< ref "/concepts/urls.md" >}}
+
+### Prefixed Buckets {#prefix}
+
+You can wrap a `*blob.Bucket` to always operate on a subfolder of the bucket
+using `blob.PrefixedBucket`:
+
+{{< goexample "gocloud.dev/blob.ExamplePrefixedBucket" >}}
+
+Alternatively, you can configure the prefix directly in the `blob.OpenBucket`
+URL:
+
+{{< goexample "gocloud.dev/blob.Example_openFromURLWithPrefix" >}}
+
+## Writing Data to a Bucket {#writing}
```

Comment:
I would suggest making a `Using` section, e.g. `## Using a Bucket`, and then moving the "Once you have opened a bucket for the storage provider you want, you can
store and access data from it using the standard Go I/O patterns." sentence from above under that. Then `### Writing` and `### Reading` are sub-heading under that...?

Reply:
Done